common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -1349,6 +1349,8 @@
   "feature.wynntils.statusEffectsOverlay.name": "Statuses",
   "feature.wynntils.statusEffectsOverlay.overlay.statusEffects.fontScale.description": "How large should the status text be?",
   "feature.wynntils.statusEffectsOverlay.overlay.statusEffects.fontScale.name": "Font Scale",
+  "feature.wynntils.statusEffectsOverlay.overlay.statusEffects.ignoreEffects.description": "Blacklist of effect you don't want to see. Separate multiple effects by adding , between them",
+  "feature.wynntils.statusEffectsOverlay.overlay.statusEffects.ignoreEffects.name": "Ignore Effects",
```

```suggestion
  "feature.wynntils.statusEffectsOverlay.overlay.statusEffects.ignoreEffects.description": "Blacklist of effects you don't want to see. Separate multiple effects by adding , between them",
  "feature.wynntils.statusEffectsOverlay.overlay.statusEffects.ignoreEffects.name": "Ignored Effects",
```

Can rename the actual config variable to ignoredEffects too if you want